steps/jandi-message/0.1.0/step.yml

```diff
@@ -0,0 +1,117 @@
+title: Send a Jandi message
+summary: Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+description: |
+  Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+  You can setup **Incoming WebHook integration**.
+  To register an **Incoming WebHook integration** visit http://bit.ly/31Bxeeh
+
+  *Don't forget to enable this Step to **Always run** if
```

Comment:
currently, it is turned on by default: `is_always_run: true`, so I would remove this sentence.